server/views/pages/induction/prePrisonEducation/qualificationsList.njk

```diff
@@ -85,7 +85,7 @@ Data supplied to this template:
                 <td class="govuk-table__cell" data-qa="educational-qualification-subject">{{ item.subject }}</td>
                 <td class="govuk-table__cell" data-qa="educational-qualification-grade">{{ item.grade }}</td>
                 <td class="govuk-table__cell">
-                  <button type="submit" class="app-u-button-as-link" value="{{ item.level }}-{{ item.subject }}-{{ item.grade }}" name="removeQualification">Remove</button>
+                  <button type="submit" class="app-u-button-as-link" value="{{ loop.index0 }}" name="removeQualification">Remove</button>
```

Comment:
`loop` is a nunjucks variable that you automatically get when within a nunjucks `for` loop
The property `index` is the 1 based index; the property `index0` is the zero based index